debian/install

```diff
@@ -0,0 +1,8 @@
+htdocs usr/share/loris/
+jsx usr/share/loris/
+modules usr/share/loris/
+smarty usr/share/loris/
+test usr/share/loris/
+php usr/share/loris/
+tools usr/share/loris/
+
```

Comment:
what's the format of this file? Is it only directories, or should it also copy the config file to /etc/loris/config.xml?